lib/client.js

```diff
@@ -357,22 +391,42 @@ extend(Raven.prototype, {
       res.sentry = eventId;
       return next(err);
     };
+  },
+
+  captureBreadcrumb: function (breadcrumb) {
+    if (!this.installed) return;
```

Comment:
Does `installed` work like it does in raven-js? (Where you can choose not to "install" raven-js but that really just affects `window.onerror` and auto instrumentation.) 

In other words – do we need this check?